aries_vcx/src/handlers/out_of_band/sender/sender.rs

```diff
@@ -54,11 +58,11 @@ impl OutOfBandSender {
 
     pub fn append_a2a_message(mut self, msg: A2AMessage) -> VcxResult<Self> {
         let (attach_id, attach) = match msg {
-            A2AMessage::PresentationRequest(request) => {
-                (AttachmentId::PresentationRequest, json!(&request).to_string())
+            a2a_msg @ A2AMessage::PresentationRequest(_) => {
+                (AttachmentId::PresentationRequest, json!(&a2a_msg).to_string())
             }
-            A2AMessage::CredentialOffer(offer) => {
-                 (AttachmentId::CredentialOffer, json!(&offer).to_string())
+            a2a_msg @ A2AMessage::CredentialOffer(_) => {
+                 (AttachmentId::CredentialOffer, json!(&a2a_msg).to_string())
```

Comment:
This is bugfix - previously the we serialized type `CredentialOffer`, not as `A2Amessage::CredentialOffer` - therefore the serialized json was missing `@type`. This is fixed now.